accessibility-checker.php

```diff
@@ -834,7 +834,13 @@ function edac_summary_ajax() {
 	$html['content'] = '';
 
 	// password check.
-	if ( boolval( get_option( 'edac_password_protected' ) ) === true ) {
+	if (
+		! (
+			EDAC_KEY_VALID === true &&
+			edac_check_plugin_active( 'accessibility-checker-pro/accessibility-checker-pro.php' )
+		) &&
+		\EDAC\Helpers::is_basic_auth() 
+	) {
```

Comment:
Only show the "Whoops! It looks like your website is currently password protected..." message when editing pages if Pro is not valid/active and we've tested that basic auth is in place.